src/parser/jobs/war/modules/InnerRelease.js

```diff
@@ -71,6 +71,7 @@ export default class InnerRelease extends Module {
 
 	_onRemoveIR() {
 		// TODO: You may need to make adjustments so the guard isn't necessary. The applybuff event is fab'd for things at the start of the fight, it may be a go.
+		/* eslint-disable-next-line no-magic-numbers */
```

Comment:
More things to fix instead of suppress. You should declare that 5 as a constant, and then you can also use it later if you so desire to display the target Fell Cleave count when generating the module output.